P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -1023,164 +1023,365 @@ overshadows the historical content</quote>.</egXML>
+            <graphic url="Images/billiardhall.jpg" width="149px" height="276px"/>
+          </figure>
+          
+          This example is intriguing in that the right-side ruby 
+          glosses apply to the first and second characters respectively, but
+          the left-side gloss applies to the whole word as a unit. We use this 
+          instance to exemplify multiple approaches to encoding the same
+          phenomena, which may be appropriate for different projects or 
+          editorial preferences. First, using the same segmentation approach 
+          as demonstrated for <mentioned>大学</mentioned> above, but 
+          with nesting:
+          
+          <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <p style="writing-mode: vertical-rl">
+              [...]
+              <ruby>
+                <rb>
+                  <ruby>
+                    <rb>打</rb>
+                    <rt place="right">ダ</rt>
+                  </ruby> 
+                  <ruby>
+                    <rb>球</rb>
+                    <rt place="right">キウ</rt>
+                  </ruby>
+                  場
+                </rb>
+                <rt place="left">ビリヤード</rt>
+              </ruby>
+              [...]
+            </p>
+          </egXML>
+          
+          We could also use a standoff approach with <gi>anchor</gi> elements and 
+          pointers:
+          
+          <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <p style="writing-mode: vertical-rl"> 
+              [...]
+              <ruby>
+                <rb>
+                  <anchor xml:id="da"/>打
+                  <anchor xml:id="kyuu"/>球
+                  <anchor xml:id="ba"/>場
+                  <anchor  xml:id="owari"/>
+                </rb>
+                <rt place="left" from="#da" to="#owari">ビリヤード</rt>
+                <rt place="right" from="#da" to="#kyuu">ダ</rt>
+                <rt place="right" from="#kyuu" to="#ba">キウ</rt>
+              </ruby> 
+              [...]
+            </p>
+          </egXML>
+          
+          Alternatively, if the encoding itself already includes segmentation below
+          the word level, we can use the existing elements instead of adding <gi>anchor</gi>s:
+          
+          <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <p style="writing-mode: vertical-rl"> 
+              [...]
+              <ruby>
+                <rb xml:id="dakyuuba"><c xml:id="c1">打</c><c xml:id="c2">球</c><c>場</c></rb>
+                <rt place="left" target="#dakyuuba">ビリヤード</rt>
+                <rt place="right" target="#c1">ダ</rt>
+                <rt place="right" target="#c2">キウ</rt>
+              </ruby> 
+              
+              [...]
+            </p>
+          </egXML>
+          
+        </p>
+        
+        <p>The current support for ruby is rudimentary, and in future releases of the
```

Comment:
I don't see anything wrong with the sentence as it stands, and since you don't think your own suggestion is right either, I don't see that it's worth changing it. That sentence is only going to be there for a single release of the Guidelines, I hope.